### ISSUE TITLE
Breadcrumb accessibility: Can't keyboard navigate with JAWS

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1595]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

With JAWS on you are unable to navigate the nodes in the breadcrumb.
Note that even without JAWS, one can't go to the overflow "..." node.

Expected: Using arrow keys will navigate all the nodes.

**Comment by betrue-final-final** [comment]
None of the above repro in Narrator. However the breadcrumb does not tell what it is (page issue), and the "..." menu doesn't have a label. Should be something like "Unshown parent folders" or something like that. We need UA support .

**Comment by nikhilexe** [comment]
I checked here https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/breadcrumb and it seems this is fixed recently but this is not working here https://dev.office.com/fabric#/components/breadcrumb. Can somebody confirm if this is already fixed?

**Comment by aadhars** [comment]
Also looks like it is fixed for chrome but not for firefox

**Comment by betrue-final-final** [comment]
This is fixed for Narrator on Chrome, Edge, and Firefox. As this is a JAWS only bug, will send the issue to our JAWS contact.
